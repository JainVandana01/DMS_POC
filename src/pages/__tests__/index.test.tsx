import { render } from "@testing-library/react";
import App from "../";

jest.mock("next/router", () => ({
  __esModule: true,
  useRouter: jest.fn(),
  push: jest.fn(),
}));

describe("App", () => {
  it("renders App unchanged", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
