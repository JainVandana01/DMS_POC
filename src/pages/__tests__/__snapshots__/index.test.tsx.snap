// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App unchanged 1`] = `
<div>
  <ul>
    <li>
      <a
        href="/login"
      >
        Login
      </a>
    </li>
    <li>
      <a
        href="/signup"
      >
        Sign Up
      </a>
    </li>
    <li>
      <a
        href="/home"
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/dashboard"
      >
        Dashboard
      </a>
    </li>
  </ul>
</div>
`;
